pine-a64-plus/scripts/setup-user.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+set -e
+
+echo "I: create user"
+useradd --comment "User" --create-home --shell /bin/bash user
+
+echo "user	ALL=NOPASSWD: ALL" >/etc/sudoers.d/user
```

Comment:
add user to sudo group instead

Reply:
tbh I'd also vote the `/etc/sudoers.d/user` file ;)
however /me would use the `overlays/` for it rather than scripting.

Reply:
I prefer to stay with actual configuration, its more convenient for me